app/js/components/homePage/manageApps/manageApps.jsx

```diff
@@ -83,8 +145,12 @@ export default class ManageApps extends React.Component {
         return {
           appList: response.data.appList,
           msgType: 'success',
-          msgBody: 'App deleted successfully.',
+          /** 
```

Comment:
I think you forgot to remove this commented code.

Reply:
@andela-jomadoye I commented on this part of code to raise an issue about a bug that arises when the delete method is called. The 'showMsg' state remains unchanging and cannot be manipulated from any other part of the Code. You can pull changes to try and verify this.

Reply:
@paulupendo did you take a look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotcommentoutcode
A ticket should be enough to remind you about the bug.

Reply:
@dkayiwa Comment removed and @andela-jomadoye has [this ticket](https://issues.openmrs.org/browse/AOM-21) regarding the bug.